dcat/index.html

```diff
@@ -133,38 +133,55 @@ <h2>Motivation for change</h2>
-        <tr><td><code>sdo</code></td><td><code>https://schema.org/</code></td></tr>
-        <tr><td><code>skos</code></td><td><code>http://www.w3.org/2004/02/skos/core#</code></td></tr>
-        <tr><td><code>time</code></td><td><code>http://www.w3.org/2006/time#</code></td></tr>
-        <tr><td><code>vcard</code></td><td><code>http://www.w3.org/2006/vcard/ns#</code></td></tr>
-        <tr><td><code>w3cgeo</code></td><td><code>http://www.w3.org/2003/01/geo/wgs84_pos#</code></td></tr>
-        <tr><td><code>xsd</code></td><td><code>http://www.w3.org/2001/XMLSchema#</code></td></tr>
-        </tbody>
-    </table>
+        DCAT also makes extensive use of terms from other vocabularies, in particular Dublin Core [[?DCTERMS]].
+        DCAT defines a minimal set of classes and properties of its own.</p>
+        <section>
+          <h3>Namespaces appearing in recommendations</h3>
+          <p>Namespaces and prefixes used in normative parts of this recommendation are shown in the following table.</p>
+
+          <table id="table-namespaces">
+            <thead><tr><th>Prefix</th><th>Namespace</th></tr></thead>
+            <tbody>
+            <tr><td><code>dc</code></td><td><code>http://purl.org/dc/elements/1.1/</code></td></tr>
+            <tr><td><code>dcat</code></td><td><code>http://www.w3.org/ns/dcat#</code></td></tr>
+            <tr><td><code>dct</code></td><td><code>http://purl.org/dc/terms/</code></td></tr>
+            <tr><td><code>dctype</code></td><td><code>http://purl.org/dc/dcmitype/</code></td></tr>
+            <tr><td><code>foaf</code></td><td><code>http://xmlns.com/foaf/0.1/</code></td></tr>
+            <tr><td><code>locn</code></td><td><code>http://www.w3.org/ns/locn#</code></td></tr>
+            <tr><td><code>odrl</code></td><td><code>http://www.w3.org/ns/odrl/2/</code></td></tr>
+            <tr><td><code>owl</code></td><td><code>http://www.w3.org/2002/07/owl#</code></td></tr>
+            <tr><td><code>prov</code></td><td><code>http://www.w3.org/ns/prov#</code></td></tr>
+            <tr><td><code>rdf</code></td><td><code>http://www.w3.org/1999/02/22-rdf-syntax-ns#</code></td></tr>
+            <tr><td><code>rdfs</code></td><td><code>http://www.w3.org/2000/01/rdf-schema#</code></td></tr>
+            <tr><td><code>skos</code></td><td><code>http://www.w3.org/2004/02/skos/core#</code></td></tr>
+            <tr><td><code>time</code></td><td><code>http://www.w3.org/2006/time#</code></td></tr>
+            <tr><td><code>vcard</code></td><td><code>http://www.w3.org/2006/vcard/ns#</code></td></tr>
+            <tr><td><code>w3cgeo</code></td><td><code>http://www.w3.org/2003/01/geo/wgs84_pos#</code></td></tr>
+            <tr><td><code>xsd</code></td><td><code>http://www.w3.org/2001/XMLSchema#</code></td></tr>
+            </tbody>
+        </table>
+    
+        </section>
+        <section class="informative">
+          <h3>Namespaces appearing in examples</h3>
```

Comment:
@dr-shorthair : A minor thing, perhaps, I would mention that the namespaces in the second table are used also in guidelines.

```suggestion
          <h3>Namespaces appearing in examples and non-normative guidelines</h3>
```